.github/workflows/release-to-pypi.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Release to PyPI
+on:
+  create:
+    tags:
+      - '*'
+  workflow_dispatch:
```

Comment:
We can keep this in for this one, but then need to get rid of it, otherwise it will double publish on certain releases and fail